.travis.yml

```diff
@@ -39,42 +39,51 @@ matrix:
     include:
 
         # Do a coverage test in Python 3.
-        - python: 3.6
+        - python: 3.7
```

Comment:
with other packages we leave it to conda to deal with the python versions (via the `PYTHON_VERSION` variable) and use `generic` or `c` as the travis language. Maybe that would solve your issue here, too?